juice-examples/mackey-glass-rnn-regression/src/main.rs

```diff
@@ -138,11 +168,11 @@ fn train(
     // Initialise a CUDA Backend, and the CUDNN and CUBLAS libraries.
     let backend = Rc::new(get_cuda_backend());
 
-    let example_count: usize = 44000;
+    let example_count = 35192;
     let columns: usize = 10;
```

Comment:
make this a constant an re-use instead of magic numbers :)

Reply:
And the `columns` too :)

Reply:
Done!